packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -2906,6 +2904,14 @@ export function reappearLayoutEffects(
         safelyAttachRef(finishedWork, finishedWork.return);
         break;
       }
+      break;
+    }
+    case Fragment: {
+      if (enableFragmentRefs) {
+        if (flags & Ref) {
```

Comment:
I think you can skip this check for now. Nothing else is doing it and I don't actually think it's safe to do it.